indico/modules/receipts/client/js/printing/TemplateParameterEditor.jsx

```diff
@@ -68,6 +82,19 @@ function CustomField({name, value, type, attributes, validations, onChange}) {
         required={validations.required}
       />
     );
+  } else if (type === 'image') {
+    return (
+      <EventImageSelectField
+        label={attributes.label}
+        name={name}
+        value={value}
+        required={validations.required}
+        loading={loadingImages}
+        onChange={onChange}
+        eventImages={eventImages}
+        onOpen={() => fetchImages()}
```

Comment:
Why do we do this every time we open it? Can't we do it just once when loading the dialog?

Reply:
I think when most people who are preparing a document to be generated won't be expecting/remember that the images have to be pre-uploaded... Imagine filling out everything and then realizing you have to do it all over again to get the new image after uploading it 😅

Reply:
--> just add a "refresh images" button somewhere that calls reFetch?